docs/apache-airflow/concepts.rst

```diff
@@ -655,11 +655,11 @@ Relationship Builders
 
 *Moved in Airflow 2.0*
 
-In Airflow 2.0 those two methods moved from ``airflow.utils.helpers`` to ``airflow.models.baseoperator``.
-
 ``chain`` and ``cross_downstream`` function provide easier ways to set relationships
```

Comment:
```suggestion
:func:`airflow.models.baseoperator.chain` and :func:`airflow.models.baseoperator..cross_downstream` function provide easier ways to set relationships
```
This will ensure easy access to the reference documentation. See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-roles